README.md

```diff
@@ -0,0 +1,39 @@
+# elastic-meetup-demo
+
+This repo contains the code needed to create a stack of elasticsearch, kibana, nats, filebeat and metricbeat.
+
+Both metricbeat and filebeat are configured to have newly introduced nats-module enabled for collecting logs and metrics
+from nats message queue respectively.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+In order to build the stack Linux OS is required and the following prerequisites need to be installed in the system.
+
+1) docker(latest version)
+2) docker-compose(latest version)
+3) golang
+4) gcc
+5) python
+
+Also user has to be member of sudoers group.
+
+### Installing
+
+```
```

Comment:
This block of code needs to break and every command should be accompanied with explanatory text. (e.g. build_filebeat is a script which create a custom Filebeat image because nats module is not yet part of Filebeat 7.0 but it will be introduced to 7.1, vp.max_map_count is required by Elasticsearch, with link to documentation https://www.elastic.co/guide/en/elasticsearch/reference/7.0/docker.html#docker-cli-run-prod-mode etc. etc.)